docker/build-tools/Dockerfile

```diff
@@ -337,7 +337,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # Docker including docker-ce, docker-ce-cli, and containerd.io
 ADD https://download.docker.com/linux/ubuntu/gpg /tmp/docker-key
 RUN apt-key add /tmp/docker-key
-RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
+RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
+# RUN apt search docker-ce docker-ce-cli containerd.io
```

Comment:
Might add a little context for this RUN line so people don't think its stray debug.

Reply:
Probably should just remove it.  I'm not sure adding it really helps.